### ISSUE TITLE
Why loss is different from the sum of the Train net output?

### ISSUE THREAD

**Issue by SingL3** [BVLC/caffe#6480]
I am training a network. And I got the log like this:

> I0725 15:24:17.703490 30674 solver.cpp:228] Iteration 540, loss = 0.0669772
I0725 15:24:17.703541 30674 solver.cpp:244]     Train net output #0: ClsLoss = 0.02642 (* 1 = 0.02642 loss)
I0725 15:24:17.703564 30674 solver.cpp:244]     Train net output #1: LossBBox = 0.00156623 (* 5 = 0.00783116 loss)
I0725 15:24:17.703573 30674 sgd_solver.cpp:106] Iteration 540, lr = 0.0025

As you can see, the loss is different from the sum of 2 outputs. I have commented the average_loss in solver, so I want to know why is that?

**Comment by shaibagon** [comment]
Please use the [caffe-users list][1] for usage, installation, or modeling questions, or other requests for help.
You may also post questions on [stackoverflow][3], make sure you tag them with `caffe` tag.  
There is also [caffe.help][4] documenting the different layers of caffe.
Do not post such requests to Issues. Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing][2] before submitting this issue.

 [1]: https://groups.google.com/forum/#!forum/caffe-users
 [2]: https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md
 [3]: http://stackoverflow.com/questions/tagged/caffe
 [4]: http://caffe.help/manual/layers.html
